pq-crypto/sike_r2/config.h

```diff
@@ -1,232 +1,222 @@
-/********************************************************************************************
-* SIDH: an efficient supersingular isogeny cryptography library
-*
-* Abstract: configuration file and platform-dependent macros
-*********************************************************************************************/
-
-#ifndef SIKE_CONFIG_H
-#define SIKE_CONFIG_H
-
-#include <stdint.h>
-#include <stdbool.h>
-#include <stddef.h>
-
-// Definition of operating system
-
-#define OS_WIN 1
-#define OS_LINUX 2
-
-#if defined(_WIN32) // Microsoft Windows OS
-#define OS_TARGET OS_WIN
-#else
-#define OS_TARGET OS_LINUX // default to Linux
-#endif
-
-// Definition of compiler (removed in OQS)
-
-#define COMPILER_GCC     1
-#define COMPILER_CLANG   2
-
-#if defined(__GNUC__)           // GNU GCC compiler
-#define COMPILER COMPILER_GCC
-#elif defined(__clang__)        // Clang compiler
-#define COMPILER COMPILER_CLANG
-#else
-#error -- "Unsupported COMPILER"
-#endif
-
-
-// TODO - Hardcoding these #defines for now to get the build working with the generic C code.
-//  Need to enable builds for multiple architectures, and enable/include optimized x64 assembly
-//  code
-#define _AMD64_
-#define _GENERIC_
-
-// Definition of the targeted architecture and basic data types
-

```

Comment:
I removed these hardcoded `#define`s, and added a comment about where we determine which code to use.